src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+/* stylelint-disable font-family-name-quotes */
+/* stylelint-disable rule-empty-line-before */
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
```

Comment:
Using the universal selector `*` can have performance implications as it applies styles to all elements. Consider using more specific selectors if possible.